WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecAttachmentViewController.swift

```diff
@@ -91,6 +99,15 @@ class AztecAttachmentViewController: UITableViewController {
 
     // MARK: - Actions
 
+    private func displayAltTextfield(row: ImmuTableRow) {
+        let editableRow = row as! EditableTextRow
+        let hint = NSLocalizedString("Image Description", comment: "Hint for image description on image settings.")
```

Comment:
Replace both occurrences of "Description" with "Alt"

Reply:
Fixed